cluster/charts/rook/templates/deployment.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
 spec:
   replicas: 1
+  selector:
```

Comment:
@mxey should we also put this selector on the `rook-operator.yaml`?

Reply:
@travisn no. 

At the API version `apps/v1beta1` that `rook-operator.yaml` uses spec.selector will default to the value of `template.metadata.labels`, which already only contains the app=rook-operator label.

The issue in the Helm chart was with the version in the label, because spec.selector is immutable on deployments.